components/Order/order_events.rst

```diff
@@ -0,0 +1,27 @@
+.. _component_order_sylius-order-events:
+
+SyliusOrderEvents
+=================
+
+.. hint::
+    To benefit from events you should implement or use the Symfony's `EventDispatcher`_.
+
+.. _EventDispatcher: http://symfony.com/doc/current/components/event_dispatcher/introduction.html
+
+Order component provide by default following events:
+
++--------------------------+------------------------------------+
+| Event                    | Description                        |
++==========================+====================================+
+| sylius.order.pre_create  | Launched before the order creation |
++--------------------------+------------------------------------+
+| sylius.order.post_create | Launched after the order creation  |
++--------------------------+------------------------------------+
+| sylius.order.pre_update  | Launched before the order updated  |
++--------------------------+------------------------------------+
+| sylius.order.post_update | Launched after the order updated   |
++--------------------------+------------------------------------+
+| sylius.order.pre_delete  | Launched before the order deletion |
++--------------------------+------------------------------------+
+| sylius.order.post_delete | Launched after the order deletion  |
++--------------------------+------------------------------------+
```

Comment:
I hate to say that, but I would remove this part of documentation for now, for all components. The reason is following: Nothing really specific is happening for those events now and they are for controller actions, which does not concern the component itself. We will be doing a lot of changes to events and this will require more work to rework existing docs than add new parts if needed.